sdks/go/test/integration/primitives/windowinto.go

```diff
@@ -110,7 +117,7 @@ func TriggerDefault(s beam.Scope) {
 
 	col := teststream.Create(s, con)
 	windowSize := 10 * time.Second
-	validate(s.Scope("Fixed"), window.NewFixedWindows(windowSize), col, window.Trigger{Kind: window.DefaultTrigger}, window.Accumulating, 6.0, 9.0)
+	validateEquals(s.Scope("Fixed"), window.NewFixedWindows(windowSize), col, window.TriggerDefault(), window.Discarding, 6.0, 9.0)
```

Comment:
Why did the accumulation mode change from Accumulating to Discarding here?

Reply:
To keep it consistent with other trigger tests. The expected output will still be the same as before because the firing is done only once at the end of window.